spanner/client_test.go

```diff
@@ -5208,7 +5208,7 @@ func TestClient_CloseWithUnresponsiveBackend(t *testing.T) {
 	server.TestSpanner.Freeze()
 	defer server.TestSpanner.Unfreeze()
 
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Millisecond)
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Millisecond)
```

Comment:
We should have tests that verify that:
1. Multiplexed sessions are used for single-use and multi-use read-only transactions (when enabled).
2. Regular sessions are used for read/write transactions (when multiplexed sessions are enabled).
3. Only one multiplexed session is created, even if there are more than 1 goroutine trying to execute a read-only transaction directly at startup before the pool has been initialized.
4. That multiplexed sessions are refreshed at the configured interval.
5. That if replacing a multiplexed session after the configured interval fails, it does not affect the existing multiplexed session. (That is: If the initial session creation succeeds, and replacing the session after 7 days fails, then the original session should continue to be used.)

Reply:
1,2,3  => Added in `TestClient_MultiplexedSession`
4,5 => Added in `TestMultiplexSessionWorker`